script/run-in-env.sh

```diff
@@ -9,9 +9,10 @@ if [ -s .python-version ]; then
 fi
 
 # other common virtualenvs
+my_path="${0%/*}/.."
```

Comment:
This assumes the user is using a bash (or a bash compatible) shell. The shebang actually is `sh`, so this will not work for those cases.